package.json

```diff
@@ -146,6 +146,7 @@
     "publish-preminor": "node scripts/check-publish-access && node scripts/clear-package-dir preminor --verbose && lerna publish preminor --pre-dist-tag=next --preid=next --force-publish --allow-branch=master --message=\"chore(release): Publish next pre-minor\"",
     "publish-rc": "node scripts/check-publish-access && node scripts/clear-package-dir prerelease --verbose && lerna publish prerelease --pre-dist-tag=rc --preid=rc --message=\"chore(release): Publish rc\"",
     "publish-release": "node scripts/check-publish-access && node scripts/clear-package-dir patch --verbose && lerna publish patch",
+    "publish-v4": "node scripts/release-cli.js publish v4",
```

Comment:
If this script sole purpose is for next major releases NIT comment I have is maybe include `v4` or `next-major` in script file name to avoid potential confusion?

Reply:
sure